Gemfile

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'activemodel'
+gem 'activesupport'
+gem 'aws-sdk-cloudformation'
+gem 'aws-sdk-ec2'
+gem 'azure_mgmt_compute'
+gem 'azure_mgmt_network'
+gem 'azure_mgmt_resources'
+gem 'colorize'
+gem 'commander', git: 'https://github.com/alces-software/commander'
+gem 'google-cloud-resource_manager'
```

Comment:
This can be dropped until we're adding GCP support.

Reply:
https://github.com/alces-software/cloudware/commit/1428d0dcfb3193212f39b0859173c84e0dfd5dbe